sormas-backend/src/main/java/de/symeda/sormas/backend/report/AggregateReportFacadeEjb.java

```diff
@@ -179,7 +178,11 @@ public List<AggregatedCaseCountDto> getIndexList(AggregateReportCriteria criteri
 		}
 
 		List<AggregatedCaseCountDto> reportList = new ArrayList<>(reportSet);
-		reportList.sort(Comparator.comparing(r -> r.getDisease().toString()));
+		Function<AggregatedCaseCountDto, String> objectObjectFunction = r -> r.getDisease().toString();
+		Comparator<AggregatedCaseCountDto> comparator = Comparator.comparing(objectObjectFunction)
+			.thenComparing(r -> r.getAgeGroup().replaceAll("\\d", ""))
```

Comment:
This fails with a NullPointerException when there are aggregate reports without age groups in the system (which is very likely)